From 1efcdf981e6f4cc4e2b5e5dfe407030dc1e43e65 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 27 Jun 2015 13:29:52 -0700
Subject: [PATCH 2/2] chromium/third_party/boringssl: Demand for newer POSIX
 macro

Reason for change: Define _POSIX_C_SOURCE such that it demands correct
posix interfaces, netdb.h declares interfaces such as
getaddrinfo if __USE_POSIX, i.e. POSIX.1:1990 or later.
However, these interfaces were new in the 2001 edition of POSIX
therefore ask for Extension from POSIX.1:2001 since we use addrinfo
structure here.

Change-Id: Icb1c92745d1a0ca958108ae80c270c630628729e
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Reviewed-on: https://boringssl-review.googlesource.com/5253
Reviewed-by: Adam Langley <agl@google.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/3rdparty/chromium/third_party/boringssl/src/crypto/bio/socket_helper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/3rdparty/chromium/third_party/boringssl/src/crypto/bio/socket_helper.c b/src/3rdparty/chromium/third_party/boringssl/src/crypto/bio/socket_helper.c
index ba65a1a..66c3976 100644
--- a/src/3rdparty/chromium/third_party/boringssl/src/crypto/bio/socket_helper.c
+++ b/src/3rdparty/chromium/third_party/boringssl/src/crypto/bio/socket_helper.c
@@ -12,7 +12,7 @@
  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
 
-#define _POSIX_SOURCE
+#define _POSIX_C_SOURCE 200112L
 
 #include <openssl/bio.h>
 #include <openssl/err.h>
-- 
2.5.3

